onnxscript/rewriter/onnxruntime/bfloat16_utils/bfloat16_converter.py

```diff
@@ -0,0 +1,84 @@
+import logging
+
+from onnxscript import ir
+
+logger = logging.getLogger(__name__)
+
+
+def _convert_inputs_from_bfloat16_to_float16(value: ir.Input):
+    if value.dtype == ir.DataType.BFLOAT16:
+        value.dtype = ir.DataType.FLOAT16
+        value_users = list(value.uses())
+        for node, index in value_users:
+            _insert_cast_nodes_for_float16_to_bfloat16_to_inputs(node, index)
+
+
+def _convert_outputs_from_bfloat16_to_float16(value: ir.Value):
+    if value.dtype == ir.DataType.BFLOAT16:
```

Comment:
```suggestion
    if value.dtype != ir.DataType.BFLOAT16:
        return
```

negate and return early to reduce nest level

Reply:
Done